libs/sdk-backend-tiger/src/backend/workspace/dashboards/index.ts

```diff
@@ -175,6 +188,28 @@ export class TigerWorkspaceDashboards implements IWorkspaceDashboardsService {
         });
     };
 
+    private getFilterContextFromExportId = async (exportId: string): Promise<IFilterContext> => {
+        const result = await this.authCall((client) => {
+            return client.export.getMetadata({
+                workspaceId: this.workspace,
+                exportId,
+            });
+        }).then((result) => result.data);
+
+        // metadata is under our control, accepts arbitrary json, currently we
+        // store there only `filters` array for the export-based filters
+        const { filters } = result as any as { filters: FilterContextItem[] };
+
+        return {
+            filters,
+            title: `temp-filter-context-${exportId}`,
+            description: "temp-filter-context-description",
```

Comment:
What are these used for? Is it ok to used `temp` in it?

Reply:
I think it is, it's just so that it fits the general type for the converter which just extracts the filters from there anyway